doc/docs/2.3.x/how-tos/jupyterlab-extension/index.md

```diff
@@ -1,173 +1,115 @@
+    ```sh 
+    Jupyter Server [...] is running at:
+    ```
+4. Navigate to the connection tab. You will need to provide a link formatted like the following:
+    ```sh
+    grpc://<cluster-ip>:<port>
+    ```
+5. Open another terminal and run the following to get the IP address and port number:
+     ```sh 
+     kubectl get services | grep -w "pachd "
+     ```
+6. Find the `servic/pachd` line item and copy the **IP address** and first **port number**.
+    
+    ```sh
+       NAME                          TYPE           CLUSTER-IP       EXTERNAL-IP   PORT
+       pachd                         ClusterIP      10.106.225.116   <none>        30650/TCP,30657/TCP,30658/TCP,30600/TCP,30656/TCP
+    ```
+7. Input the full connection URL (`grpc://10.106.225.116:30650`).
+8. Navigate to the **Launcher** view in Jupyter and select **Terminal**.
+9. Input the following command:
+    ```sh
+    pachctl version
+    ```
+10. If you see a `pachctl` and `pachd` version, you are good to go.
 
-    # This sets the user back to the notebook user account (i.e., Jovyan) 
-    USER $NB_UID
+### Install to Existing Docker Image 
 
-    # Replace the version here with the version of the extension you would like to install from https://pypi.org/project/jupyterlab-pachyderm/ 
-    RUN pip install jupyterlab-pachyderm==<version> 
-    ```
+Replace the following `${PACHCTL_VERSION}` with the version of `pachctl` that matches your cluster's, and update `<version>` with the release number of the extension.
 
-    Then, [build, tag, and push your image](../developer-workflow/working-with-pipelines/#step-2-build-your-docker-image).
+You can find the latest available version of our Pachyderm Mount Extension in [PyPi](https://pypi.org/project/jupyterlab-pachyderm/). 
 
-### Running the JupyterLab Container
+```sh
+# This runs the following section as root; if adding to an existing Dockerfile, set the user back to whatever you need. 
+USER root
 
-If you are using our pre-built image:
+# This is the directory files will be mounted to, mirroring how pipelines are run. 
+RUN mkdir -p /pfs 
 
-- Run the script below:
+# If you are not using "jovyan" as your notebook user, replace the user here. 
+RUN chown $NB_USER /pfs
 
-    ```shell
-    docker run -it -p 8888:8888 -e GRANT_SUDO=yes --user root --device /dev/fuse --privileged --entrypoint /opt/conda/bin/jupyter pachyderm/notebooks-user:{{ config.jupyterlab_extension_image_tag }} lab --allow-root 
-    ```
+# Fuse is a requirement for the mount extension 
+RUN apt-get clean && RUN apt-get update && apt-get -y install curl fuse 
 
-- Click on the link provided in the stdout of your terminal to run JupyterLab in a browser.
+# Install Pachctl - Set the version of Pachctl that matches your cluster deployment. 
+RUN curl -f -o pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v${PACHCTL_VERSION}/pachctl_${PACHCTL_VERSION}_amd64.deb 
+RUN dpkg -i pachctl.deb
```

Comment:
Additionally, add instructions for installing the mount-server binary here

Reply:
thanks for the link! I've added the install steps and updated the pachctl verbiage to include "optionally"